peekingduck/nodes/draw/bbox.py

```diff
@@ -36,24 +36,52 @@ class Node(AbstractNode):
 
         |bbox_labels_data|
 
+        |bbox_scores_data|
```

Comment:
Realised PeekingDuck does not document *optional inputs*, like the `mot.metadata` in `model.fairmot`.
Would suggest changing above to `|bbox_scores_data| (optional input)` to make it clearer to the user.

Reply:
Resolved